Ginger/GingerCoreNET/Reports/RunsetFromReportLoader.cs

```diff
@@ -29,25 +29,37 @@ public RunsetFromReportLoader()
 
         public async Task<RunSetConfig?> LoadAsync(RunSetReport runsetReport)
         {
-            string runsetName = runsetReport.Name;
-            RunSetConfig? runset = GetRunsetFromSolutionRepository(runsetName);
+            string executionId = runsetReport.GUID;
+            Guid runsetId;
+            if (executionId.Contains('-'))
+            {
+                runsetId = Guid.Parse(executionId);
+            }
+            else
+            {
+                runsetId = Guid.ParseExact(executionId, "N");
```

Comment:
For LiteDB executions, RunSet GUID and Execution ID will not be the same, still it will load the RunSet?

Reply:
made necessary changes to handle this